control_panel_api/management/commands/add_oidc_statement_to_users_roles.py

```diff
@@ -0,0 +1,66 @@
+import json
+import logging
+
+import boto3
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from control_panel_api.models import User
+
+
+OIDC_PROVIDER_ARN = f'{settings.IAM_ARN_BASE}:oidc-provider/{settings.OIDC_DOMAIN}/'
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Adds OIDC statement to all users roles trust policies"
+
+    def handle(self, *args, **options):
+        iam = boto3.resource('iam')
+        for user in User.objects.all():
+            # Read role trust policy
+            iam_role = iam.Role(user.iam_role_name)
+            assume_role_policy_document = iam_role.assume_role_policy_document
```

Comment:
I usually take comments to indicate that there should be a self-documenting function instead, eg:
```python
for user in User.objects.all():
    policy = read_role_trust_policy(user)

    if has_oidc_statement(policy):
        logger.info('...Skipping.')

    else:
        add_oidc_statement(policy, user)
        update_role_trust_policy(policy, user)
        logger.info('OIDC statement added ...')
```

Reply:
I take your point. One reason why I didn't go that route is to avoid making this too opaque.

It's relatively straightforward and I didn't want to over-engineer it but at the same time leave some crumbs behind for the future me/others (therefore the couple of comments).